mvt/provider.go

```diff
@@ -1,11 +1,20 @@
 package mvt
 
-import "github.com/terranodo/tegola"
+import (
+	"context"
 
-//Provider is the mechinism by which the system talks to different data providers.
+	"github.com/terranodo/tegola"
+)
+
+var ErrCanceled = context.Canceled
+
+//Provider is the mechanism by which the system talks to different data providers.
 type Provider interface {
 	// MVTLayer returns a layer object based
 	MVTLayer(layerName string, tile tegola.Tile, tags map[string]interface{}) (*Layer, error)
```

Comment:
We may want to remove this definition in the Interface. I left it in for backward compatibility, but am now thinking this is a mistake. I think it makes more sense to just have people use the MVTLayerWithContext.